javascript/ql/src/Security/CWE-078/IndirectCommandInjection.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Indirect uncontrolled command line
+ * @description Using command-line arguments to construct other
+ *              command-line arguments may indirectly introduce command-line
```

Comment:
Perhaps clearer: "Forwarding command-line arguments to a child process executed within a shell may indirectly (...)"